PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -193,6 +201,9 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
         mvaFall17V1noIso = cms.InputTag("electronMVAValueMapProducer:ElectronMVAEstimatorRun2Fall17NoIsoV1Values"),
         mvaFall17V2Iso = cms.InputTag("electronMVAValueMapProducer:ElectronMVAEstimatorRun2Fall17IsoV2Values"),
         mvaFall17V2noIso = cms.InputTag("electronMVAValueMapProducer:ElectronMVAEstimatorRun2Fall17NoIsoV2Values"),
+        # mvaIso = cms.InputTag("electronMVAValueMapProducer:ElectronMVAEstimatorRun2RunIIIWinter22IsoV1Values"),
```

Comment:
If there is a special purpose, or you may need these lines in future then keep it. Otherwise, delete this line.